staff/martin-garcia/staff/PasapalabraHTML/work.js

```diff
@@ -0,0 +1,298 @@
+            addToRanking();
+            mostrarRanking();
+            cambiarDescripcion("reiniciar");
+            clearInterval(intervalo);
+        }
+    }, 1000);
+}
+
+
+function cambiarDescripcion(juego) {
+    if (juego === "inicio") {
+        var div = document.getElementById("inicio");
+        div.hidden = true;
+        div = document.getElementById("reiniciar");
+        div.hidden = true;
+        div = document.getElementById("inGame");
+        div.hidden = false;
+    } else if (juego === "reiniciar") {
+        var div = document.getElementById("inGame");
+        div.hidden = true;
+        div = document.getElementById("reiniciar");
+        div.hidden = false;
+    }
+}
+
+
+
+
+function formularPregunta(tipo) {
+    var textBox = document.getElementById("respuesta");
+    textBox.value = "";
+    textBox.focus();
+    if (tipo === "PrimeraRonda") {
+        var div = document.getElementById("pregunta");
+        div.innerHTML = questions[validacion.contadorVuelta].question;
+        tipoPregunta = "PrimeraRonda";
+        comprobacionPregunta = questions[validacion.contadorVuelta].question;
+        comprobacionRespuesta = questions[validacion.contadorVuelta].answer;
+    } else if (tipo === "NoContestada") {
+        if (numeroPreguntaNoContestada === noContestadas.length || numeroPreguntaNoContestada < 0) {
+            numeroPreguntaNoContestada = 0;
+        }
+        var div = document.getElementById("pregunta");
+        div.innerHTML = noContestadas[numeroPreguntaNoContestada].question;
+        tipoPregunta = "NoContestada";
+        comprobacionPregunta = noContestadas[numeroPreguntaNoContestada].question;
+        comprobacionRespuesta = noContestadas[numeroPreguntaNoContestada].answer;
+    }
+
+}
+
+
+function responder() {
+    if (tipoPregunta === "PrimeraRonda") {
+        var respuesta = document.getElementById("respuesta").value;
+        validarPregunta(questions, respuesta, validacion.contadorVuelta);
+    } else {
+        var respuesta = document.getElementById("respuesta").value;
+        validarPregunta(noContestadas, respuesta, numeroPreguntaNoContestada);
+        console.log("if " + noContestadas[numeroPreguntaNoContestada].status);
+        if (noContestadas[numeroPreguntaNoContestada].status == 1) {
+            console.log("Correcta");
+            questions[questions.indexOf(noContestadas[numeroPreguntaNoContestada])].status == 1;
+            noContestadas.splice(numeroPreguntaNoContestada, 1);
+            numeroPreguntaNoContestada -= 1;
+        } else if (noContestadas[numeroPreguntaNoContestada].status == -1) {
+            console.log("False");
+            questions[questions.indexOf(noContestadas[numeroPreguntaNoContestada])].status == -1;
+            noContestadas.splice(numeroPreguntaNoContestada, 1);
+            numeroPreguntaNoContestada -= 1;
+        } else {
+            numeroPreguntaNoContestada += 1;
+        }
+    }
+    validacion.contadorVuelta += 1;
```

Comment:
This function can be more simple with a switch case :)

Reply:
Yes, thx for the tip, It's already solved